marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx

```diff
@@ -89,7 +89,9 @@ describe('Create/Edit Mapping Step artifact component', () => {
     const queryInput = getByPlaceholderText('Enter Source Query');
     fireEvent.change(queryInput, { target: {value: 'cts.collectionQuery(["testCollection"])'}});
     expect(queryInput).toHaveTextContent('cts.collectionQuery(["testCollection"])');
-
+    fireEvent.change(queryInput, { target: {value: 'cts.collectionQuery("testCollection")'}});
+    expect(queryInput).toHaveTextContent('cts.collectionQuery("testCollection")');
```

Comment:
I think a test is needed for cts.collectionQuery('testCollection') as well, as single quotes are valid. 

There's also nothing prohibiting a user from having e.g. cts.collectionQuery(["one", "two"]) . I am assuming that if that's the case, then ["one", "two"] will show up?

Reply:
@rjrudin when user has cts.collectionQuery(["one", "two"]) then it collection Name shows as one,"two. Since in order to fetch collection name we are grabbing characters between initial start and end quotes.